tools/test-case-generators/format-request-map.json

```diff
@@ -885,6 +991,81 @@
           }
         }
       },
+      "rhel-84": {
```

Comment:
I have the same comment as for the `fedora-36` override... Was this needed?

Reply:
Same error as in fedora 36